src/content/docs/en/welcome.mdx

```diff
@@ -76,6 +76,46 @@ As of now, the only data we collect is a count of how many times a package has b
+
+However, Ultramarine Linux includes Terra, RPM Fusion and Flathub.
+- Terra is an RPM repository provided by Fyra Labs (the team behind Ultramarine Project), and it mainly uses a rolling release scheme.
+- RPM Fusion is another (very popular) third party repository for Fedora. It uses a stable release scheme.
+- Flathub is a cross-distro Flatpak repository. Updates in this repository has nothing to do with Fedora/Ultramarine's release cycles. Strictly speaking, packages in Flathub are rolling because they are updated once upstream releases a version.
+
+### If some packages are updated mid-release, doesn't that mean Ultramarine is unstable/insecure compared to, e.g. Debian?
+
+**No**. That's an incorrect approach to view if a distribution is secure/stable or not.
+- Packages in Fedora/Ultramarine are updated mid-release, but those updates **only include non-breaking changes**. Your computer has virtually a near-zero chance of breaking mid-release due to an update.
+- Undeniably, Ultramarine and Fedora has much more frequent updates compared to distros like Debian, and to some extent, they might be more unstable. However, Fedora/Ultramarine's release cycles make it so that the operating system is still stable enough for most normal uses. Millions of Fedora devices across the world run fine without breaking, which shows Fedora's high stability. Debian's release cycle is also arguably unnecessarily too slow and can cause more hassles than necessary.
+- Debian does not provide updates that are already available on Fedora for months. Users on Debian might miss out a lot, including new features, bug fixes, security patches, etc. This means Debian could be even more insecure than Fedora/Ultramarine.
+
+### Why Ultramarine (instead of other distros like Ubuntu)
+
+- performance
+  - Fedora (the distro Ultramarine is based on) has one of the best IO performance over other distros like Ubuntu.
+  - BTRFS is the default file system, a relatively high-performant file system with features like snapshots. You can backup your files in less than a second (unlike other file systems like ext4).
+  - We include lightweight desktop environments like Budgie (Flagship Edition) and XFCE. They look great, work well and are snappy.
+- Privacy
+  - We do not collect your private data ever because that's a waste of storage.
+  - We don't even know how many Ultramarine devices there are (unlike some companies).
+- Security
+  - Security updates are provided quickly (unlike some other distros).
+  - SELinux is active by default.
+- Customisations
+  - There are multiple desktop environments for you to choose from.
+  - We have applets installed out of the box that help you with further desktop customizations.
+- Usability
+  - You are not required to ever touch the command line.

```

Comment:
wording is a tad funny here, maybe frame it more as lack of need and not lack of requirement